src/eos-dropbox-app

```diff
@@ -111,6 +111,10 @@ class DropboxLauncher():
         self._quit_if_name_lost = False
         self._daemon = None
 
+        # For communication with the OpenURI portal only
+        self._bus = None
+        self._proxy = None
```

Comment:
These two fields are now only referenced in `_open_dropbox_directory` so they should be locals.

Reply:
Oops. Right, I did this because of a previous iteration of the PR, but it's no longer needed now